### DISCUSSION TITLE
assignAutomerge doesn't seem to work

### DISCUSSION THREAD

**Discussion by bavedarnow** [renovatebot/renovate#8920] [Request Help] [answered]
```js
{
...
  "assignAutomerge": true,
...
  "packageRules": [
    {
      "matchPackageNames": ["package1", "package2"],
      "matchUpdateTypes": ["patch"],
      "automerge": true
    }
  ]
}
```

With this config, the Assign Automerge feature doesn't seem to work. Does it need to be scoped within the package rules?

**Comment by rarkins** [comment, marked as answer]
It doesn't need to be in the same package rule. Can you create a minimal reproduction which demonstrates the problem? Eg one dependency, one automerging PR, no assignee. 

Note that the decision whether to add an assignee is done at Pr creation only. It won't assign existing PRs due to this setting.

**Reply by bavedarnow** [reply]
> Note that the decision whether to add an assignee is done at Pr creation only. It won't assign existing PRs due to this setting.

This is likely why it wasn't working. I'll check it the next time a PR is opened and mark this answer as solved
